go.mod

```diff
@@ -1,22 +1,26 @@
 module github.com/nginxinc/nginx-ingress-operator
 
-go 1.16
+go 1.15
```

Comment:
is changing 15 to 16 intended?

Reply:
it wasn't supported, it is now with 1.8.0